sonar-plugin-api/src/main/java/org/sonar/api/batch/sensor/internal/SensorContextTester.java

```diff
@@ -142,11 +144,21 @@ public SensorContextTester setActiveRules(ActiveRules activeRules) {
    */
   @Override
   public Version getSonarQubeVersion() {
-    return sqVersion.get();
+    return sqVersion.getApiVersion();
   }
 
-  public SensorContextTester setSonarQubeVersion(Version version) {
-    this.sqVersion = new SonarQubeVersion(version);
+  @Override
+  public Version getRuntimeApiVersion() {
+    return sqVersion.getApiVersion();
+  }
+
+  @Override
+  public SonarProduct getRuntimeProduct() {
+    return sqVersion.getProduct();
+  }
+
+  public SensorContextTester setRuntime(Version version, SonarProduct product, SonarQubeSide sonarQubeSide) {
+    this.sqVersion = new SonarQubeVersion(version, product, sonarQubeSide);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this use of "SonarQubeVersion"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)